### PR TITLE
Reduce LDC diff a bit

### PR DESCRIPTION
**Pull request by kinke**

### CODE REVIEW

File: src/rt/dwarfeh.d [hunk 1]
```diff
@@ -473,26 +575,25 @@ ClassInfo getClassInfo(_Unwind_Exception* exceptionObject, const(ubyte)* current
...
+            debug (EH_personality) writeln("break: %p %p", currentLsd, ehn.languageSpecificData);
             break;
         }
 
-        //printf("ehn =   %p '%.*s'\n", ehn.object, cast(int)ehn.object.classinfo.info.name.length, ehn.object.classinfo.info.name.ptr);
+        debug (EH_personality) writeln("ehn =   %p '%.*s'", ehn.object, cast(int)ehn.object.classinfo.info.name.length, ehn.object.classinfo.info.name.ptr);
         Error e = cast(Error)ehobject;
         if (e is null || (cast(Error)ehn.object) !is null)
         {
-            currentLsd = ehn.languageSpecificData;
```
Comment (kinke) [inline src/rt/dwarfeh.d:491]: This was a no-op - `currentLsd` is checked for being equal to `ehn.languageSpecificData` a few lines up.

File: src/rt/aaA.d [hunk 2]
```diff
@@ -533,12 +533,16 @@ extern (C) void* _aaGetY(AA* aa, const TypeInfo_AssociativeArray ti,
...
  */
-extern (C) void* _aaGetX(AA* aa, const TypeInfo_AssociativeArray ti,
+extern (C) void* _aaGetX(AA* paa, const TypeInfo_AssociativeArray ti,
     const size_t valsz, scope const void* pkey, out bool found)
 {
     // lazily alloc implementation
-    if (aa.impl is null)
-        aa.impl = new Impl(ti);
+    AA aa = *paa;
+    if (aa is null)
```
Comment (ibuclaw) [inline src/rt/aaA.d:541]: Why this?

**Review by ibuclaw — APPROVED** [review]

File: src/rt/aaA.d (hunk 2 above)
Comment (kinke) [reply src/rt/aaA.d:541]: What do you mean? If `AA == Impl*`, there's of course no `.impl`.

Comment (ibuclaw) [reply src/rt/aaA.d:541]: You mean you're just taking advantage of the `alias this`?

Comment (kinke) [reply src/rt/aaA.d:541]: Yes. The `paa` to `aa` deref is required, because previous `paa.firstUsed` was a double-deref ('paa->impl->firstUsed').

**Pull request merged**